### PR TITLE
DTD-2820 cannotFindLockReasonTrue test

### PR DESCRIPTION
**Pull request by rogerball**

### CODE REVIEW

File: conf/resources/data/sa/cannotFindLockReasonTrue.json
Comment (paigedellafranca) [inline conf/resources/data/sa/cannotFindLockReasonTrue.json:1]: is "string" ok to fill fields with under the "addresses" array or do we need examples like the "individual" array has?

Comment (rogerball) [reply conf/resources/data/sa/cannotFindLockReasonTrue.json:1]: A good question. We use "string" 673 times in the stubs so I repeated the same here. There isn't a value to verifying those fields in every test. However, if there is a good argument then we can create a story to  change them all from "string".

Comment (paigedellafranca) [reply conf/resources/data/sa/cannotFindLockReasonTrue.json:1]: ok thanks for the info!

**Review by paigedellafranca — APPROVED** [review]
changes made match acceptance criteria for DTD-2820

**Pull request merged**